hadoop-tools/hadoop-azure/src/site/markdown/testing_azure.md

```diff
@@ -935,3 +935,30 @@ http[s]://[account][domain-suffix]/[filesystem], please use the following:
   <value>{IP}:{PORT}</value>
 </property>
 ```
+
+##Run different combinations of tests using the runtests.sh script
```

Comment:
if this is going to be the policy. pull it up to the top of the document after the "no test no review" textHey